service/host.go

```diff
@@ -278,9 +285,12 @@ func (s *host) start() Control {
 
 				s.shutdownMu.Unlock()
 				if _, err := s.shutdown(e, "", nil); err != nil {
-					s.Logger().Error("Unable to shut down modules", "initialError", e, "shutdownError", err)
+					s.Logger().Error("Unable to shut down modules",
+						zap.String("initialError", e.Error()),
+						zap.String("shutdownError", err.Error()),
```

Comment:
We're going to lose the cause/stacktrace of these errors by turning them into Strings, right?